_ja/overviews/scala3-migration/tooling-tour.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 移行ツールのツアー
+type: chapter
+description: この章は移行ツールのエコシステムのツアーです
+num: 6
+previous-page: compatibility-metaprogramming
+next-page: tooling-migration-mode
+language: ja
+---
+
+## Scalaコンパイラ
+
+移行に関して、簡単でスムーズに行えるように、事前に2つのコンパイラが準備している。
```

Comment:
```suggestion
移行が簡単でスムーズになるように、2系と3系両方コンパイラにおいて周到にマイグレーションの準備が進められてきた。
```

原文: The migration has been carefully prepared beforehand in each of the two compilers so that the transition is easy and smooth.